src/mono/wasm/templates/templates/wasi-console/wasi-console.0.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netX.0</TargetFramework>
```

Comment:
net8 ? should we update the browser templates to net8 too ?

Reply:
`netX.0` is just the string that will get replaced by the actual tfm used.